modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/timers/select-reassignment/SelectReassignment.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayForm, {ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+import SidebarPanel from '../../../SidebarPanel';
+
+const options = [
```

Comment:
@eliasantosf You could move this array to an external file and import it here and in the `SelectAssignment` file, since they use the same options.

Reply:
I suggest opening a technical debt in the future for this change, because the options are not exactly the same, the last element of the Array is different. Therefore, changing only the last element of this Array at the moment will bring more difficulties to deliveries, as other functions must be used to change the Array value in certain sections.